### ISSUE TITLE
screen record audio

### ISSUE THREAD

**Issue by RustoMCSpit** [FossifyOrg/Voice-Recorder#73]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Voice-Recorder/issues) or [closed](https://github.com/FossifyOrg/Voice-Recorder/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Voice-Recorder/discussions) or [closed](https://github.com/FossifyOrg/Voice-Recorder/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Voice-Recorder/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

i would like to record the audio on my phone and also id like to be able to record it and talk over it.

basically, record the software on my phone and me talking over it

### Why do you want this feature?

so i can review things more easily for my business

### Additional information

somewhat similar https://github.com/FossifyOrg/Voice-Recorder/issues/70

**Comment by HT-7** [comment]
Agreed. Windows and Linux could do this for decades.

**Comment by RustoMCSpit** [comment]
Yes.

**Comment by RustoMCSpit** [comment]
Just to be clear, the external mic should only try pick up on external sounds, like how on call with someone there's no echo.

**Comment by naveensingh** [comment]
Yeah, I think that would be a separate app. I have one, I might rework it into Fossify Screen Recorder some day.
